open-api.yaml

```diff
@@ -2363,16 +2428,82 @@ paths:
               schema:
                 type: object
                 properties:
-                  private:
-                    type: string
-                  public:
-                    type: string
-                additionalProperties: false
-                required:
-                  - private
-                  - public
+                  results:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/key'
         '401':
           $ref: '#/components/responses/401'
+  '/keys/{key}':
+    get:
+      summary: Get an API key
+      tags:
+        - Keys
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/key'
+      operationId: keys.get
+      description: Get an API Key
+    parameters:
+      - schema:
+          type: string
+        name: key
+        in: path
+        required: true
+        description: The API key value
+    delete:
+      summary: Delete an API key
+      operationId: keys.delete
+      tags:
+        - Keys
+      responses:
+        '204':
+          $ref: '#/components/responses/204'
+      description: ''
+    patch:
```

Comment:
Are we sure we use patch instead of put since it's not consistent with the other route?

Reply:
Yes, I forgot to update the spec for that but we should use `patch` to be in accordance with the use of HTTP verbs. As we're going to pass on to most of the HTTP routes before 1.0, we'll have to update them when we can. We could also do this at once to correct the use of verbs already.